src/cmake-tools.ts

```diff
@@ -504,9 +504,13 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
         const quickStart = localize('quickstart.cmake.project', "Create");
         const changeSourceDirectory = localize('edit.setting', "Locate");
         const ignoreCMakeListsMissing = localize('ignore.activation', "Don't show again");
-        const showCMakeLists: boolean = await expShowCMakeLists();
 
-        telemetryProperties["missingCMakeListsPopupType"] = showCMakeLists ? "selectFromAllCMakeLists" : "toastCreateLocateIgnore";
+        const showCMakeLists: boolean = await expShowCMakeLists();
+        const existingCmakeListsFiles: string[] | undefined = await util.getAllCMakeListsPaths(this.folder.uri);
+        telemetryProperties["missingCMakeListsPopupType"] = (showCMakeLists && existingCmakeListsFiles && existingCmakeListsFiles.length > 0) ? "selectFromAllCMakeLists" : "toastCreateLocateIgnore";
+        if (showCMakeLists) {
+          telemetryProperties["ignoreExperiment"] = (!existingCmakeListsFiles || existingCmakeListsFiles.length === 0).toString();
```

Comment:
`showCMakeLists` should also be updated based on the value of `existingCmakeListsFiles.length`

Reply:
good catch, thx. I just pushed an update.